product-releases/2021-05-03-falco-support.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Support for Falco
+author: Daniel McSheehy
+author_title: Software Engineer
+author_url: https://github.com/DanielMcSheehy
+author_image_url: https://avatars.githubusercontent.com/u/28493438?s=460&u=4bbd1de69b9ca85d21e45350d15db419a27072fc&v=4
+tags: [security]
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Last week, the effx platform engineering team landed support for [Falco](https://falco.org/).
+
+Falco is a popular open-source security and compliance tool.
+In fact, it's listed as the [defacto Kubernetes threat detection engine](https://falco.org/).
+The Falco integration displays real-time vulnerabilities when microservices do not meet compliance.
+
+[Falco]: https://falco.org/
+
+<!--truncate-->
+
+View security alerts, and vulnerabilities for all your services.
+
+<p>
+  <img alt="feed screenshot"
+      src={useBaseUrl('img/integrations/falco-feed.png')}/>
+</p>
+
+Set a security standard to make sure your services are always compliant.
+
+<p>
+  <img alt="feed screenshot"
+      src={useBaseUrl('img/integrations/falco-standard.png')}/>
+</p>
+
+Curious about how it works?
+[Sign up](http://app.effx.com/sign_up) today and give it a try!
```

Comment:
nit, but since we have the links below, we don't have to put it in parentheses here